src/driver/aurora-data-api/AuroraDataApiDriver.ts

```diff
@@ -583,6 +583,40 @@ export class AuroraDataApiDriver implements Driver {
         }
     }
 
+    /**
+     * Normalizes "setDefault" value of the column.
+     */
+     normalizeSetDefault(columnMetadata: ColumnMetadata): string | undefined {
+      const defaultValue = columnMetadata.setDefault;
+
+      if (defaultValue === null) {
+          return undefined;
+      }
+
+      if ((columnMetadata.type === "enum" || columnMetadata.type === "simple-enum") && defaultValue !== undefined) {
+          return `'${defaultValue}'`;
+      }
+      if ((columnMetadata.type === "set") && defaultValue !== undefined) {
+          return `'${DateUtils.simpleArrayToString(defaultValue)}'`;
+
+      }
+      if (typeof defaultValue === "number") {
+          return "" + defaultValue;
+
+      } else if (typeof defaultValue === "boolean") {
+          return defaultValue === true ? "1" : "0";
+
+      } else if (typeof defaultValue === "function") {
+          return defaultValue();
```

Comment:
Good idea, will add that.

Reply:
I think properly evaluating a promise here would require too large of a change to everything that leads to this function (i.e. I cant use async/await here, if there is another method of evaluating a promise synchronously please let me know I'm not aware). We can also just return an error message that a promise is not supported, or default to NULL (or database DEFAULT)

Reply:
> We can also just return an error message that a promise is not supported

If a lot of changes have to be involved this could be an alternative solution.

In my opinion default to NULL seems to be not the ideal solution because this might bring debugging issue/confusion to users.

Reply:
Did some work towards this but was unable to find a good way to detect both promises and functions that return promises in either code or via types. The existing `default` field doesn't fail gracefully either if you try to use a promise. Anyone wanna take a crack at this part for both options?